src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -249,3 +250,35 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
 
     with self.argument_context('acr check-health') as c:
         c.argument('ignore_errors', options_list=['--ignore-errors'], help='Provide all health checks, even if errors are found', action='store_true', required=False)
+
+    with self.argument_context('acr scope-map') as c:
+        c.argument('registry_name', options_list=['--registryName', '-r'])
```

Comment:
Should be `--registry-name`. 
Also for non create command, please support [ids parameters](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-the-ids-parameter)

Reply:
We have changed it to `--registry`. `--registry` command line arg is used across other places in ACR.